BDDTest/Features/CatalanNumbers.feature

```diff
@@ -0,0 +1,17 @@
+﻿Feature: CatalanNumbers
+	Simple calculator for adding two numbers
+
+@getRange
+Scenario: Get range of Catalan numbers
+	When I get a range of catalan numbers from 0 to 25
+	Then the range should be correct
```

Comment:
I think it would be more expressive, to list the actual expected values here instead of just writing "correct".

Reply:
Ok but than is it okay if I only test for a smaller range? The Catalan numbers become long pretty fast and I was afraid the test would become unreadable.

Reply:
Indeed, the list would be too long. Testing a smaller range is a good idea then.